ruby/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'mutant',             '~> 0.7.5', git: 'https://github.com/mbj/mutant.git'
```

Comment:
@filipvanlaenen There is a bug in the last mutant release that can cause the reporter to crash, its fixed in master. I'll make a release before monday, expect a followup PR to bump the version here. (Maybe give me commit access to the repo).